coinstacks/bitcoin/ingester/src/parseTx/index.ts

```diff
@@ -23,23 +23,23 @@ export const parseTx = async (tx: Tx, address: string): Promise<BTCParseTx> => {
     send: {},
   }
 
-  tx.vin.forEach((vin: Vin) => {
+  tx.vin.forEach((vin) => {
     if (vin.isAddress === true && vin.addresses?.includes(address)) {
       // send amount
       const sendValue = new BigNumber(vin.value ?? 0)
       if (!sendValue.isNaN() && sendValue.gt(0)) {
         pTx.send['BTC'] = aggregateTransfer(pTx.send['BTC'], sendValue.toString(10))
+      }
 
-        // network fee (only for sends)
-        const fees = new BigNumber(tx.fees ?? 0)
-        if (!fees.isNaN() && fees.gt(0)) {
-          pTx.fee = { symbol: 'BTC', value: fees.toString(10) }
-        }
+      // network fee
+      const fees = new BigNumber(tx.fees ?? 0)
+      if (!fees.isNaN() && fees.gt(0)) {
```

Comment:
`fees.gt(0)` will return `false` if `fees.isNaN()` is true, so you don't need to do both checks.

in web we use this helper:
```ts
export const bnOrZero = (n: BigNumber.Value | null | undefined): BigNumber => {
  const value = new BigNumber(n || 0)
  return value.isNaN() ? bn(0) : value
}
```